### ISSUE TITLE
Duplicate alleles?

### ISSUE THREAD

**Issue by kellrott** [bmeg/bmeg-etl#320]
Doing a test load, using the manifest from `s3://bmeg/bmeg_rc1_2/bmeg.manifest` and the file base as `s3://bmeg/bmeg_rc1_2/`, I get a bunch of `duplicate key error collection`

```
Allele:000a410a71666a3fe4385d78959a4efc95507809
Allele:001a6071f1d0c995a0d1b73f101078e08a2a16c0
Allele:001e196197dfd89812fe5cdfaad1681a86a7b669
```

**Comment by bwalsh** [comment]
```
buntu@neo4j-test:/mnt/bmeg/bmeg-etl/outputs/allele$ zcat Allele.Vertex.json.gz | grep Allele:000a410a71666a3fe4385d78959a4efc95507809
{"_id": "Allele:000a410a71666a3fe4385d78959a4efc95507809", "gid": "Allele:000a410a71666a3fe4385d78959a4efc95507809", "label": "Allele", "data": {"genome": "GRCh37", "chromosome": "17", "start": 74039963, "end": 74039963, "strand": "+", "reference_bases": "T", "alternate_bases": "T", "hugo_symbol": "SRP68", "ensembl_transcript": "ENST00000307877", "type": "SNP", "effect": "Missense_Mutation"}}
{"_id": "Allele:000a410a71666a3fe4385d78959a4efc95507809", "gid": "Allele:000a410a71666a3fe4385d78959a4efc95507809", "label": "Allele", "data": {"genome": "GRCh37", "chromosome": "17", "start": 74039963, "end": 74039963, "strand": "+", "reference_bases": "T", "alternate_bases": "T", "hugo_symbol": "SRP68", "ensembl_transcript": "ENST00000307877", "type": "SNP", "effect": "Nonsense_Mutation"}}
ubuntu@neo4j-test:/mnt/bmeg/bmeg-etl/outputs/allele$ zcat Allele.Vertex.json.gz | grep Allele:001a6071f1d0c995a0d1b73f101078e08a2a16c0
{"_id": "Allele:001a6071f1d0c995a0d1b73f101078e08a2a16c0", "gid": "Allele:001a6071f1d0c995a0d1b73f101078e08a2a16c0", "label": "Allele", "data": {"genome": "GRCh37", "chromosome": "17", "start": 66992119, "end": 66992119, "strand": "+", "reference_bases": "C", "alternate_bases": "C", "hugo_symbol": "ABCA9", "ensembl_transcript": "ENST00000340001", "type": "SNP", "effect": "Missense_Mutation"}}
{"_id": "Allele:001a6071f1d0c995a0d1b73f101078e08a2a16c0", "gid": "Allele:001a6071f1d0c995a0d1b73f101078e08a2a16c0", "label": "Allele", "data": {"genome": "GRCh37", "chromosome": "17", "start": 66992119, "end": 66992119, "strand": "+", "reference_bases": "C", "alternate_bases": "C", "hugo_symbol": "ABCA9", "ensembl_transcript": "ENST00000340001", "type": "SNP", "effect": "Nonsense_Mutation"}}
ubuntu@neo4j-test:/mnt/bmeg/bmeg-etl/outputs/allele$ zcat Allele.Vertex.json.gz | grep Allele:001e196197dfd89812fe5cdfaad1681a86a7b669
{"_id": "Allele:001e196197dfd89812fe5cdfaad1681a86a7b669", "gid": "Allele:001e196197dfd89812fe5cdfaad1681a86a7b669", "label": "Allele", "data": {"genome": "GRCh37", "chromosome": "17", "start": 37881332, "end": 37881332, "strand": "+", "reference_bases": "G", "alternate_bases": "A", "hugo_symbol": "ERBB2", "ensembl_transcript": "ENST00000269571", "type": "SNP", "effect": "Missense_Mutation"}}
{"_id": "Allele:001e196197dfd89812fe5cdfaad1681a86a7b669", "gid": "Allele:001e196197dfd89812fe5cdfaad1681a86a7b669", "label": "Allele", "data": {"genome": "GRCh37", "chromosome": "17", "start": 37881332, "end": 37881332, "strand": "+", "reference_bases": "G", "alternate_bases": "A", "hugo_symbol": "ERBB2"}}
ubuntu@neo4j-test:/mnt/bmeg/bmeg-etl/outputs/allele$
ubuntu@neo4j-test:/mnt/bmeg/bmeg-etl/outputs/allele$
ubuntu@neo4j-test:/mnt/bmeg/bmeg-etl/outputs/allele$
ubuntu@neo4j-test:/mnt/bmeg/bmeg-etl/outputs/allele$
ubuntu@neo4j-test:/mnt/bmeg/bmeg-etl/outputs/allele$
ubuntu@neo4j-test:/mnt/bmeg/bmeg-etl/outputs/allele$ zcat Allele.Vertex.json.gz | grep Allele:001e196197dfd89812fe5cdfaad1681a86a7b669
{"_id": "Allele:001e196197dfd89812fe5cdfaad1681a86a7b669", "gid": "Allele:001e196197dfd89812fe5cdfaad1681a86a7b669", "label": "Allele", "data": {"genome": "GRCh37", "chromosome": "17", "start": 37881332, "end": 37881332, "strand": "+", "reference_bases": "G", "alternate_bases": "A", "hugo_symbol": "ERBB2", "ensembl_transcript": "ENST00000269571", "type": "SNP", "effect": "Missense_Mutation"}}
{"_id": "Allele:001e196197dfd89812fe5cdfaad1681a86a7b669", "gid": "Allele:001e196197dfd89812fe5cdfaad1681a86a7b669", "label": "Allele", "data": {"genome": "GRCh37", "chromosome": "17", "start": 37881332, "end": 37881332, "strand": "+", "reference_bases": "G", "alternate_bases": "A", "hugo_symbol": "ERBB2"}}
ubuntu@neo4j-test:/mnt/bmeg/bmeg-etl/outputs/allele$
```

**Comment by adamstruck** [comment]
What is the status of this issue? Looks like the allele normalizer needs to be updated to dedup these cases and keep the record with more information.

**Comment by adamstruck** [comment]
Fixed in #358
